src/event/sys/windows/parse.rs

```diff
@@ -97,6 +97,11 @@ fn parse_key_event_record(key_event: &KeyEventRecord) -> Option<KeyEvent> {
                     let c = character_raw as u8;
                     if c <= b'\x1F' {
                         character = (c | b'\x40') as char;
+                        // if we press something like ctrl-g, we will get `character` with value `G`.
+                        // in this case, convert the `character` to lowercase `g`.
+                        if character.is_ascii_uppercase() {
```

Comment:
This would leave capitalize char when `SHIFT + CTRL` is pressed. 

```suggestion
                        if character.is_ascii_uppercase() && !modifiers.contains(KeyModifiers::SHIFT) {
```